dask_cuda/proxy_object.py

```diff
@@ -21,6 +21,23 @@
 from .get_device_memory_objects import get_device_memory_objects
 from .is_device_object import is_device_object
 
+try:
+    from dask.array.backends import concatenate_lookup, einsum_lookup, tensordot_lookup
+    from dask.dataframe.backends import (
```

Comment:
`group_split_dispatch` and some others too seems to be present in `dask.dataframe.core` in old and new version of code. Should we just import it from there?

Reply:
Good point. Updated below using modules that appear to work in both cases for all of these